RPGSystem/Form1.Designer.cs

```diff
@@ -29,12 +29,222 @@ protected override void Dispose(bool disposing)
         private void InitializeComponent()
         {
             this.components = new System.ComponentModel.Container();
+            this.menuStrip1 = new System.Windows.Forms.MenuStrip();
```

Comment:
Names of controls or components used in the application should describe purpose of them.